doc/content/guides/getting-started/configuration.md

```diff
@@ -70,82 +71,100 @@ stack:
-TTN_LW_HTTP_COOKIE_BLOCK_KEY=... # generate 32 bytes (openssl rand -hex 32)
-TTN_LW_HTTP_METRICS_PASSWORD=... # choose a password
-TTN_LW_HTTP_PPROF_PASSWORD=...   # choose a password
-```
-
-{{% tts %}} sends emails to users, so we need to configure how those are sent. 
-
-```bash
-TTN_LW_IS_EMAIL_SENDER_NAME="{{% tts %}}"
-TTN_LW_IS_EMAIL_SENDER_ADDRESS="noreply@thethings.example.com"
-TTN_LW_IS_EMAIL_NETWORK_CONSOLE_URL="https://thethings.example.com/console"
-TTN_LW_IS_EMAIL_NETWORK_IDENTITY_SERVER_URL="https://thethings.example.com/oauth"
-```
-
-You can either use Sendgrid or an SMTP server. If you don't set up an email provider, {{% tts %}} will print emails to the server log.
+- The databases used by the stack. We will set these to the CockroachDB and Redis
+  instances that were defined in `docker-compose.yml`
+- TLS with Let's Encrypt. Since we're deploying {{% tts %}} on
+  `thethings.example.com`, we configure it to only request certificates for that
+  host, and also to use it as the default host.
+- HTTP server keys for encrypting and verifying cookies, as well as passwords
+  for endpoints that you may want to keep for internal use.
+- {{% tts %}} sends emails to users, so we need to configure how these are sent.
+  You can use Sendgrid or an SMTP server. If you skip setting up an email provider,
+  {{% tts %}} will print emails to the stack logs.
+- The URLs for the Web UI
+- The secret used for the console client.
 
-```bash
-TTN_LW_IS_EMAIL_PROVIDER="sendgrid"
-TTN_LW_IS_EMAIL_SENDGRID_API_KEY=... # enter Sendgrid API key
-```
-
-or
-
-```bash
-TTN_LW_IS_EMAIL_PROVIDER="smtp"
-TTN_LW_IS_EMAIL_SMTP_ADDRESS=...  # enter mail server address
-TTN_LW_IS_EMAIL_SMTP_USERNAME=... # enter username
-TTN_LW_IS_EMAIL_SMTP_PASSWORD=... # enter password
-```
-
-Finally, we need to configure the Web UI to use `thethings.example.com`.
-
-```bash
-TTN_LW_IS_OAUTH_UI_CANONICAL_URL="https://thethings.example.com/oauth"
-TTN_LW_IS_OAUTH_UI_IS_BASE_URL="https://thethings.example.com/api/v3"
+Below is an example `ttn-lw-stack.yml` file.
 
-TTN_LW_CONSOLE_OAUTH_AUTHORIZE_URL="https://thethings.example.com/oauth/authorize"
-TTN_LW_CONSOLE_OAUTH_TOKEN_URL="https://thethings.example.com/oauth/token"
-
-TTN_LW_CONSOLE_UI_CANONICAL_URL="https://thethings.example.com/console"
-TTN_LW_CONSOLE_UI_IS_BASE_URL="https://thethings.example.com/api/v3"
-TTN_LW_CONSOLE_UI_GS_BASE_URL="https://thethings.example.com/api/v3"
-TTN_LW_CONSOLE_UI_NS_BASE_URL="https://thethings.example.com/api/v3"
-TTN_LW_CONSOLE_UI_AS_BASE_URL="https://thethings.example.com/api/v3"
-TTN_LW_CONSOLE_UI_JS_BASE_URL="https://thethings.example.com/api/v3"
-TTN_LW_CONSOLE_UI_EDTC_BASE_URL="https://thethings.example.com/api/v3"
-TTN_LW_CONSOLE_UI_QRG_BASE_URL="https://thethings.example.com/api/v3"
-
-TTN_LW_CONSOLE_OAUTH_CLIENT_ID="console"
-TTN_LW_CONSOLE_OAUTH_CLIENT_SECRET=... # choose or generate a secret
+```yaml
+---
```

Comment:
?

Reply:
YAML syntax for document begin, not required, removed